frontend/src/styles/ProfilePage.css

```diff
@@ -48,14 +53,19 @@
 
 .header-username {
     font-size: 45px;
-    font-weight: 100;
-    margin: 0;
+    font-weight: 200;
+    margin: 2;
```

Comment:
You're defining margin as well as margin left here, it's redundant. Also add units to margin